AUTHORS.md

```diff
@@ -152,5 +152,8 @@ sravya91
 srcopela
 tobadavid
 vfrancesmolla
+Y Chandukrishna
+Josy P Pullockara
+T N Venkatesh
```

Comment:
Alphabetical order please, which in this list happens to put upper case before lower case.